src/utils/clusterUtils.js

```diff
@@ -133,3 +136,31 @@ export const clusterNodePools = (nodePools, cluster) => {
 
   return [];
 };
+
+// computeCapabilities takes a release version and provider and returns a
+// capabilities object with the features that this cluster supports.
+export function computeCapabilities(releaseVersion, provider) {
+  const capabilities = {
+    canInstallApps: false,
+    hasOptionalIngress: false,
+  };
+
+  // Installing Apps
+  // Must be AWS or KVM and larger than 8.1.0
+  // or any provider and larger than 8.2.0
+  if (
```

Comment:
I was also checking this (and other places where release versions were compared) out and thought why not drop this stuff if no one uses old versions any more (See https://github.com/giantswarm/giantswarm/blob/master/ops-recipes/list-tenant-clusters.md). The oldest tenant cluster version I found was `8.2.0`

Reply:
Yeah if the version doesn't exist anymore, we can totally drop the logic that supports it.

Reply:
alright. I am investigating cluster-service not returning enough releases im my development setup which does not let me create clusters through happa, which also uses one of these comparisons for a very old version (6). Ill see later if you have already removed these here..